cyences_app_for_splunk/default/savedsearches.conf

```diff
@@ -2170,23 +2170,26 @@ relation = greater than
 cron_schedule = 59 * * * *
 description = This alert will show the change/update in Policy. \
 \
-Data Collection - Office 365 management activity data (Splunk Add-on for Office 365). 
+Data Collection - azure:aad:audit logs from Splunk Add on for Microsoft Azure (https://splunkbase.splunk.com/app/3757)
 dispatch.earliest_time = -4h@h
 dispatch.latest_time = +2h@h
 display.general.type = statistics
 display.page.search.tab = statistics
 display.page.search.mode = fast
 request.ui_dispatch_app = cyences_app_for_splunk
 request.ui_dispatch_view = search
-search = `cs_o365_managementactivity_change("Policy")` _index_earliest=-61m@m _index_latest=-1m@m \
-| `cs_o365_managementactivity_change_out` \
+search = `cs_azure_ad_changes("Policy")` _index_earliest=-61m@m _index_latest=-1m@m \
+| rename initiatedBy.user.userPrincipalName as Actor, activityDisplayName as Command, targetResources{}.displayName as Target_DisplayName, result as ResultStatus, targetResources{}.modifiedProperties{}.displayName as displayName, targetResources{}.modifiedProperties{}.newValue as newValue, targetResources{}.modifiedProperties{}.oldValue as oldValue, additionalDetails{}.key as key, additionalDetails{}.value as value, targetResources{}.id as Target_ObjectID \
 | `cs_o365_policy_change_internal_filter` \
-| eval cyences_severity = case( in(Target_DisplayName, `cs_ad_important_policy`) or in(Target_DisplayName, `cs_ad_important_policy_internal`), "medium", true(), "low")  \
-| `cs_o365_policy_change_filter`  \
-| table event_time, cyences_severity, Actor, Command, Target_ObjectID, Target_DisplayName, ResultStatus, ModifiedProperties, MetaData
+| eval cyences_severity = case( in(Target_DisplayName, `cs_ad_important_policy`) or in(Target_DisplayName, `cs_ad_important_policy_internal`), "medium", true(), "low") \
+| eval additional_details = mvzip(key, value, " = ") \
+| table _time, cyences_severity, Actor, Command, Target_ObjectID, Target_DisplayName, displayName, oldValue, newValue, ResultStatus, modified_properties, additional_details \
+| azureadmodifiedpropertiesformatter old_value_field=oldValue new_value_field=newValue property_name_field=displayName field_to_update=modified_properties additional_fields=state \
```

Comment:
Define severity based on state field change. If state field got changed, severity should be **critical** or **high** depending on number of results daily, if it is not more than 7 times in a week change of state, we can go with critical.

Reply:
FYI, this alert scheduled hourly. 
didn't get this point => if it is not more than 7 times in a week change of state, we can go with critical.

Added commit with rest of the changes.